contrib/ec2/deis.template.json

```diff
@@ -298,6 +298,12 @@
             "Protocol": "HTTP"
           },
           {
+            "InstancePort": "443",
+            "InstanceProtocol": "HTTPS",
+            "LoadBalancerPort": "443",
+            "Protocol": "HTTPS"
+          },
```

Comment:
Maybe the instance ports here should be `80` and `HTTP`? I mean, if you're looking to terminating the HTTPS chain there then back-end routing should be HTTP